### DISCUSSION TITLE
Add new parameter to allow selection of defaults-group-suffix

### DISCUSSION THREAD

**Discussion by flybyray** [ansible-collections/ansible.mysql#457] [Ideas]
The [config_file](https://docs.ansible.com/ansible/latest/collections/community/mysql/mysql_user_module.html#parameter-config_file) can have multiple `client` entries.

Is it possible to add an optional parameter to select a specific client group by using `defaults-group-suffix` option?

See `https://dev.mysql.com/doc/refman/8.0/en/option-file-options.html#option_general_defaults-group-suffix` .
Maybe it is possible to temporary create a cnf file by extracting the slected group into new file with a rewrite of the group name to `[client]`.

**Comment by Andersson007** [comment]
@flybyray thanks for opening the discussion!
The modules use Python connectors. I don't see the opportunity to pass this flag to the connector https://pymysql.readthedocs.io/en/latest/modules/connections.html, only the path to a my.cnf file. So I don't think it's possible.
